float32 marker_id
float32 marker_size
float32 marker_center_x
float32 marker_center_y
float32 markersCount
float32 imageRows
float32 imageCols


